x-pack/plugins/timelines/server/search_strategy/timeline/factory/events/all/query.events_all.dsl.ts

```diff
@@ -46,7 +47,8 @@ export const buildTimelineEventsAllQuery = ({
     return [];
   };
 
-  const filter = [...filterClause, ...getTimerangeFilter(timerange), { match_all: {} }];
+  const filters = [...filterClause, ...getTimerangeFilter(timerange), { match_all: {} }];
+  const filter = authFilter != null ? [...filters, { ...authFilter }] : filters;
```

Comment:
nit -> do you need the spread on authFilter
```suggestion
  const filter = authFilter != null ? [...filters, authFilter] : filters;
```

Reply:
Guess we don't need to worry about mutations to authFilter here so no. I think I'm in the habit of using the spread when returning or assigning values to not copy by reference, but fine removing that here.